cmd/fluxd/main.go

```diff
@@ -278,7 +279,7 @@ func main() {
 	}
 
 	// Indirect reference to a daemon, initially of the NotReady variety
```

Comment:
fwiw, this doesn't seem like much of an indirect reference

Reply:
aw come on github, show me more than three lines of context!